openvdb/tools/ParticlesToLevelSet.h

```diff
@@ -492,7 +494,9 @@ struct ParticlesToLevelSet<SdfGridT, AttributeT, InterrupterT>::Raster
                 mParent.mInterrupter->start(
                     "Rasterizing particles to level set using const spheres");
             }
-            mTask = boost::bind(&Raster::rasterFixedSpheres, _1, _2, SdfT(radius));
+            mTask = [radius](Raster* raster, const tbb::blocked_range<size_t>& range) {
+                return raster->rasterFixedSpheres(range, SdfT(radius));
```

Comment:
To be a pure refactor, shouldn't this capture the result of `SdfT(radius)` so `SdfT` isn't called in every call of `mTask`? That is, `mTask = [SdFTOfRadius = SdfT(radius)](Raster* ...`

Reply:
Yeah good point :)